data/companies/theguardian/data.yaml

```diff
@@ -56,3 +56,6 @@ company:
           commuting_emissions_mt_per_employee_per_month: 0.040
           travel_emissions_mt_per_employee_per_month: 0.137
           revenue_allocation_to_ads_pct: 33.3
+      -
+          corporate_emissions_g_per_impression: 0.000001778
```

Comment:
When I run this locally I get `corporate_emissions_per_impression: 1.777946` ? 

```
➜ [Sep 20, 202210:33AM] ~/scope3/methodology git:(scop-211) python3 scope3_methodology/model_corporate_emissions.py publisher data/companies/theguardian/data.yaml 
corporate_emissions_mt_co2e_per_month: 794.5304000000001
➜ [Sep 20, 202210:34AM] ~/scope3/methodology git:(scop-211) ✗ python3 scope3_methodology/model_publisher_emissions.py data/companies/theguardian/data.yaml --corporateEmissionsG 794530400
properties:
- !!python/object:publisher.model.ModledProperty
  ad_revenue_allocation_pct: 23.309999999999995
  client_device_emissions_g_co2e_per_imp: 0.07246991860308909
  corporate_emissions_per_impression: 1.777946
  data_transfer_electricity_kwh: 1.0112473060344829e-05
  identifier: theguardian.com
  impressions: 10416800000.0
  page_load_electricity_kwh: 0.00012434006609195403

```

Reply:
I think we should stick to grams across the board, right now only corproate emissions is in mt. Everything else was in grams, so let's convert corporate to MT as well. Will make things easier to keep all within the same units. 

Nice call out about ad revenue allocation, found the bug. Was 
```self.corporate_emissions_g_co2e_per_impression = (
                emissions_g * self.ad_revenue_allocation_pct  / self.impressions
            )
```
should be 
```
self.corporate_emissions_g_co2e_per_impression = (
                emissions_g * self.get_ad_revenue_allocation_rate() / self.impressions
            )
```

Reply:
That being said, we do get a lot of input units in mt. So only going to change the outputs to `g` The inputs will stay in `mt` unless we also want to change that ?

Reply:
agree - mt in, g out